.github/workflows/deploy.yml

```diff
@@ -7,7 +7,6 @@ on:
 jobs:
   update_version:
     runs-on: ubuntu-latest
-    if: github.event.pull_request.merged == true
```

Comment:
I think we may still need this condition since pull requests can be closed without merging into the master branch.